src/components/FirstTimeUserFlow/FirstTimeUserFlow.js

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { useState, useEffect, createContext } from 'react';
+import { useRouter } from 'next/router';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import PersonalInfoForm from './PersonalInfoForm';
+import RoommatePrefForm from './RoommatePrefForm';
+import PropertyPrefForm from './PropertyPrefForm';
+import { postUser } from '@/firebase/userDb';
+
+const steps = ['Personal Information', 'Roommate Preference', 'Property Preference'];
+
+function getStepContent(step) {
+	switch (step) {
+		case 0:
+			return <PersonalInfoForm onFirstNameChange />;
+		case 1:
+			return <RoommatePrefForm />;
+		case 2:
+			return <PropertyPrefForm />;
+		default:
+			throw new Error('Unknown step');
+	}
+}
+
+export default function FirstTimeUserFlow() {
+	const router = useRouter();
+	const [activeStep, setActiveStep] = React.useState(0);
+	const [firstName, setFirstName] = useState('');
+	const [lastName, setLastName] = useState('');
+	const [gender, setGender] = useState('');
+	const [personInfo, setPersonInfo] = useState({});
+
+	const handleNext = () => {
+		setActiveStep(activeStep + 1);
+	};
+
+	const handleBack = () => {
+		setActiveStep(activeStep - 1);
+	};
+
+	const handleSubmit = () => {
+		const person = {
+			'first name': firstName,
+			'last name': lastName,
+		};
+		// postUser('user');
+		console.log('person', person);
```

Comment:
```suggestion
```

this can be a group discussion, but i think the log on `main` should be kept clean. obviously useful for development, but they should be deleted before merged into main. this means that the only logs that show up during development are the ones *you* personally put there during development, instead of us gradually accumulating a ton of console.logs. we could add a CI check for that.

*or* we can just give up and let the console.logs reign, it's not a big deal either way

Reply:
Agree. Removed all console. It would be great if CI can check that as well.